### PR TITLE
fix for able to make package

### PR DESCRIPTION
**Pull request by CupIvan**

### CODE REVIEW

File: archlinux/PKGBUILD [hunk 1]
```diff
@@ -12,21 +12,19 @@ depends=('mbedtls')
...
 md5sums=('SKIP')
 
 install="kadnode.install"
 
 build() {
-	cd $srcdir/KadNode
-	make FEATURES="tls bob cmd lpd nss"
+	cd "${srcdir}/KadNode"
+	make clean
+	make FEATURES="dns lpd tls bob cmd nss"
```
Comment (mwarning) [inline archlinux/PKGBUILD]: The DNS interface is not needed on arch linux. The NSS interface is used instead.

File: archlinux/PKGBUILD [hunk 2]
```diff
@@ -12,21 +12,19 @@ depends=('mbedtls')
 
 backup=('etc/kadnode/kadnode.conf' 'etc/kadnode/peers.txt')
 
-#source=(https://github.com/mwarning/KadNode.git)
-#source=(https://github.com/mwarning/KadNode/archive/v${pkgver}.tar.gz)
-source=(git+file:///home/mwarning/myProjects/KadNode)
-
+source=(git+https://github.com/mwarning/KadNode.git)
```
Comment (mwarning) [inline archlinux/PKGBUILD]: Can we keep a commented line that uses the current local files? It's nice to keep it around for testing. :-)

File: archlinux/PKGBUILD (hunk 1 above)
Comment (mwarning) [reply archlinux/PKGBUILD]: It's not a big issues, since the interface is only locally available. I guess depends on the arch linux policy.

**Comment by mwarning** [comment]
Thanks for helping out!

**Comment by CupIvan** [comment]
it's strange, but now it makes without errors with you original PKGBUILD..
so this pull request you may to delete
yesterday I make package to AUR repository, but it will be more right, if you will be maintainer :-)
https://aur.archlinux.org/packages/kadnode-git/

**Comment by mwarning** [comment]
Hi, thank you for submitting the package to Aur!
I don't mind if you keep being the maintainer. Less work for me. :-)

Anyway, maybe you like to rename the package to 'kadnode' and use the release version, instead of 'kadnode-git'. The git repository code might be broken sometimes.

**Comment by mwarning** [comment]
ok, I will close this PR und apply some small things by hand.

**Comment by mwarning** [comment]
Some changes applied by hand now: https://github.com/mwarning/KadNode/commit/862024342fd3533e450382274694c41bb738f042

**Pull request closed without merging**